src/components/ui/PopOver/index.tsx

```diff
@@ -33,7 +33,9 @@ export interface PopOverProps extends HTMLAttributes<HTMLSpanElement> {
   /** Defines the placement of PopOver menu. */
   placement?: Placement;
   /** Defines the function to render button(s). */
-  renderButton: (isActive: boolean) => {};
+  renderButton?: (isActive: boolean) => {};
+  /** Alternative to renderButton, lets you control the HTML button */
+  renderButtonWrapper?: (isActive: boolean, props: any) => {};
```

Comment:
Open to alternative naming here. Ideally it would be `renderButtonContents` and `renderButton`, but that would be a breaking change sadly

Reply:
I tried this out, by modifying the story locally to set a IconButton using RenderButtonWrapper and it worked as expected. (I had to remember to pass the props down to the IconButton)